openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -1097,3 +1098,38 @@ def verify_if_more_than_n_percentage_pod_restarted(
             msg = "Out of {} pods {} pods restarted {}".format(
                 len(pods_new), len(pods_restart), pods_restart)
             raise AssertionError(msg)
+
+    def validate_glusterfsd_pod_memory(self, size_limit):
```

Comment:
Keep this function name as generic as it can be executable on converged as well as on independent mode. The word "pod" can be removed. I will suggest something like this: 
def validate_glusterfsd_memory_usage (self, size_limit):

Reply:
Done